set.mm

```diff
@@ -522751,6 +522816,135 @@ Real and complex numbers (cont.)
       df-pred ) BCDBCEAFZGHIZJKZABLBCSMZJKZABLABCNUCUAABUBTJBCSROPQ $.
   $}
 
+
+  ${
+    $d A x $.  $d A y $.  $d B x $.  $d B y $.  $d B z $.  $d ph x $.
+    $d x y $.  $d x z $.  $d y z $.
+    $( A version of ~ ralxp with explicit substitution.  (Contributed by Scott
+       Fenton, 21-Aug-2024.) $)
+    ralxpes $p |- ( A. x e. ( A X. B ) [. ( 1st ` x ) / y ].
+     [. ( 2nd ` x ) / z ]. ph <-> A. y e. A A. z e. B ph ) $=
+      ( cv c2nd cfv wsbc c1st nfsbc1v nfcv nfsbcw nfv sbcopeq1a ralxpf ) ADBGZH
+      IZJZCRKIZJABCDEFTCUALTDCUADUAMADSLNABOACDRPQ $.
+  $}
+
+  $( Ordered triple membership in a triple cross product.  (Contributed by
+     Scott Fenton, 21-Aug-2024.) $)
+  otelxp $p |- ( <. <. A , B >. , C >. e. ( ( D X. E ) X. F ) <->
+  ( A e. D /\ B e. E /\ C e. F ) ) $=
+    ( cop cxp wcel wa w3a opelxp anbi1i df-3an 3bitr4i ) ABGZDEHZIZCFIZJADIZBEI
+    ZJZSJPCGQFHITUASKRUBSABDELMPCQFLTUASNO $.
```

Comment:
[df-ot](https://us.metamath.org/mpeuni/df-ot.html) exists, so these theorems could conflict in the future (especially ot1std and ot2ndd), although df-ot isn't used very much.

Running `show usage df-ot /rec` I get `Statement "df-ot" directly or indirectly affects the proofs of 774 statements`, about 600 or so coming from theorems that use [ot1stg](https://us.metamath.org/mpeuni/ot1stg.html)

Perhaps the name could be "o3" or "ot2"? Not sure, perhaps others have some ideas.

Reply:
Done - I decided on ot2* for these theorems